### PR TITLE
Feature/pulsar: Add tenant namespace and backoff doc 

### PR DESCRIPTION
**Pull request by Taction**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Add tenant namespace and backoff doc for pulsar.
- contrib pr dapr/components-contrib/pull/1283

## Issue reference

Please reference the issue this PR will close: #1915

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md [hunk 1]
```diff
@@ -24,14 +24,36 @@ spec:
...
+| backOffPolicy              | N        | Retry policy, `"constant"` is a backoff policy that always returns the same backoff delay. `"exponential"` is a backoff policy that increases the backoff period for each retry attempt using a randomization function that grows exponentially. Defaults to `"constant"`. | `constant`、`exponential` |
+| backOffDuration            | N        | The fixed interval only takes effect when the policy is constant. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that will be processed as milliseconds. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Defaults to `"5s"`. | `"5s"`、`"5000"`                  |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]:
```suggestion
| backOffDuration            | N        | The fixed interval only takes effect when the `backOffpolicy` is `"constant"`. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that is processed as milliseconds. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Defaults to `"5s"`. | `"5s"`、`"5000"`                  |
```

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md [hunk 2]
```diff
@@ -24,14 +24,36 @@ spec:
...
+| backOffInitialInterval     | N        | The backoff initial interval on retry. Only takes effect when the policy is exponential. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that will be processed as milliseconds. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Defaults to `"500"`                         | `"50"`                       |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]:
```suggestion
| backOffInitialInterval     | N        | The backoff initial interval on retry. Only takes effect when the `backOffpolicy` is `"exponential"`. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that is processed as milliseconds. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Defaults to `"500"`                         | `"50"`                       |
```

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md [hunk 3]
```diff
@@ -24,14 +24,36 @@ spec:
...
+| backOffMaxInterval         | N        | The backoff initial interval on retry. Only takes effect when the policy is exponential. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that will be processed as milliseconds. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Defaults to `"60s"`     | `"60000"`                     |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]:
```suggestion
| backOffMaxInterval         | N        | The backoff initial interval on retry. Only takes effect when the `backOffpolicy` is `"exponential"`. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that is processed as milliseconds. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Defaults to `"60s"`     | `"60000"`                     |
```

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md [hunk 4]
```diff
@@ -24,14 +24,36 @@ spec:
...
+| backOffMaxRetries          | N        | The maximum number of retries to process the message before returning an error. Defaults to `"0"` which means the component will not retry processing the message. `"-1"` will retry indefinitely until the message is processed or the application is shutdown. Any positive number is treated as the maximum retry count. | `"3"` |
+| backOffRandomizationFactor | N        | Randomization factor, between 1 and 0, including 0 but not 1. Randomized interval = RetryInterval * (1 ± backOffRandomizationFactor). Defaults to `"0.5"`.                 | `"0.5"`                       |
+| backOffMultiplier          | N        | Backoff multiplier for the policy. Increments the interval by multiplying it with the multiplier. Defaults to `"1.5"`         | `"1.5"`      |
+| backOffMaxElapsedTime      | N        | After MaxElapsedTime the ExponentialBackOff returns Stop. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that will be processed as milliseconds. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Defaults to `"15m"` | `"15m"` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]:
```suggestion
| backOffMaxElapsedTime      | N        | After MaxElapsedTime the ExponentialBackOff returns Stop. There are two valid formats, one is the fraction with a unit suffix format, and the other is the pure digital format that is processed as milliseconds. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Defaults to `"15m"` | `"15m"` |
```

**Review by msfussell — APPROVED** [review]
LTGM. @Taction - Added minor edits to the text. Please review and an accept the changes

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md (hunk 1 above)
Comment (Taction) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]: done.

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md (hunk 2 above)
Comment (Taction) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]: done.

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md (hunk 4 above)
Comment (Taction) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]: done.

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md (hunk 3 above)
Comment (Taction) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md]: done.

**Comment by willtsai** [comment]
@Taction - please follow the instructions [here](https://github.com/dapr/docs/pull/1930/checks?check_run_id=4632398729) to sign your commit to unblock this PR for merge. Thanks!

**Comment by Taction** [comment]
@willtsai DCO fixed.

**Comment by Taction** [comment]
@msfussell @willtsai Kindly ask for a review for this, the corresponding pr has been merged.

**Pull request merged**